server/src/test/scala/org/broadinstitute/dsp/workbench/welder/server/ObjectServiceSpec.scala

```diff
@@ -0,0 +1,63 @@
+package org.broadinstitute.dsp.workbench.welder
+package server
+
+import java.io.File
+import java.nio.file.Paths
+
+import cats.effect.IO
+import _root_.io.chrisdavenport.log4cats.Logger
+import _root_.io.chrisdavenport.log4cats.slf4j.Slf4jLogger
+import io.circe.{Json, parser}
+import fs2.io
+import org.broadinstitute.dsde.workbench.google2.Generators._
+import org.broadinstitute.dsde.workbench.google2.mock.FakeGoogleStorageInterpreter
+import org.http4s.circe.CirceEntityEncoder._
+import org.http4s.{Method, Request, Status, Uri}
+import org.scalatest.FlatSpec
+
+
+import scala.concurrent.ExecutionContext.global
+
+class ObjectServiceSpec extends FlatSpec with WelderTestSuite {
+  implicit val unsafeLogger: Logger[IO] = Slf4jLogger.getLogger[IO]
+
+  val objectService = ObjectService(FakeGoogleStorageInterpreter, global, Paths.get("fakePath"))
+  "ObjectService" should "be able to localize a file" in {
```

Comment:
Please add these test cases:
* Localize: Google Object not found
* Delocalize (positive)
* Delocalize: Google Object not found
* Delocalize: file is outdated
* Metadata (positive for each SyncStatus)
* Metadata: No storage link found
* Metadata: Google Object not found
* Metadata: Google Object Metadata is OK but we can't retrieve the object itself

Reply:
thanks @jmthibault79 for reminder.

I think lots of the failure path errors are still tbd, so I'll add doc for those first before I add tests since likely expected behavior will change